services/flows-operator/src/FlowOperator.js

```diff
@@ -58,24 +58,24 @@ class FlowOperator {
                 await this._undeployJob({
                     metadata: {
                         name: node.metadata.name
-                    } 
-                });  
+                    }
+                });
```

Comment:
plz add rule for line ending into eslintrc

Reply:
@anton-kotenko sorry, what rule do you mean?